recipe/meta.yaml

```diff
@@ -1,34 +1,117 @@
 {% set name = "numpyro" %}
-{% set version = "0.11.0" %}
+{% set version = "0.15.3" %}
 
 package:
-  name: numpyro
+  name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: d357dd1a0159f86d45c233708ad33a3d3d53431d05b4bbece1d068ae4168aa8e
+  #url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  url: https://github.com/pyro-ppl/{{ name }}/archive/refs/tags/{{ version }}.tar.gz
+  sha256: b8fbcc259575dfb9068e947015acde42b389772141a0d24acc640d2335e16d99
 
 build:
-  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
   number: 0
-  script: {{ PYTHON }} -m pip install .
+  # Missing jax and jaxlib python version 3.9
+  # https://github.com/jax-ml/jax/blob/jax-v0.4.35/setup.py#L118
+  # Missing jax and jaxlib for s390x
+  # The numpyro max version python 3.11
```

Comment:
Will it work with py312? Or do some of the dependencies have constraints?

Reply:
Only 3.9-3.11

Reply:
I see 3.9-3.11 tagged in `setup.py`, but the actual constraint is `python_requires=">=3.9"`. Might be worth at least attempting to build for 3.12 unless there's some specific dependency constraint.

Reply:
@Andrii-Osipov can we try to build for py312 as well before?